packages/account/src/Components/poi/idv-document-submit/idv-document-submit.tsx

```diff
@@ -11,13 +11,25 @@ import {
     filterObjProperties,
     isDesktop,
     removeEmptyPropertiesFromObject,
+    formatIDVFormValues,
 } from '@deriv/shared';
 import { documentAdditionalError, getRegex, validate, makeSettingsRequest, validateName } from 'Helpers/utils';
 import FormFooter from 'Components/form-footer';
 import BackButtonIcon from 'Assets/ic-poi-back-btn.svg';
 import IDVForm from 'Components/forms/idv-form';
 import PersonalDetailsForm from 'Components/forms/personal-details-form';
 import FormSubHeader from 'Components/form-sub-header';
+import { GetSettings, IdentityVerificationAddDocumentResponse } from '@deriv/api-types';
+import { TIDVFormValues } from 'Types';
+
+type TIdvDocumentSubmitProps = {
```

Comment:
sometime you capitalise IDV and sometime you miss on that, keep it consistent please 🙏🏼

Reply:
Will fix this